<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIAbout</class>
 <widget class="QDialog" name="UIAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="windowTitle about">About : %1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/about/about.png</normaloff>:/about/about.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lLogoS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/about/images/about.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lInformations">
       <property name="text">
        <string>&lt;strong&gt;%1 version %2&lt;/strong&gt;&lt;br/&gt;(Build for %3)&lt;br/&gt;&lt;a href=&quot;%4&quot;&gt;Raptor's Home page&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="twAbout">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QLabel" name="lLogo">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources.qrc">:/about/raptor.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <attribute name="title">
       <string comment="tab about">About</string>
      </attribute>
     </widget>
     <widget class="QTextBrowser" name="tbTeam">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string comment="tab team">Team</string>
      </attribute>
     </widget>
     <widget class="QTextBrowser" name="tbLicense">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="tabStopWidth">
       <number>20</number>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string comment="tab license">License</string>
      </attribute>
     </widget>
     <widget class="QTextBrowser" name="tbDonations">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; color:#000000; background-color:#ffffff;&quot;&gt;If you like Raptor and you want to help make it even more awesome : As you know, time is priceless and if you have some to spare then there is no better way to help an open source project than to get involved at one level or another. Contributions are appreciated and needed in any of the following ways:&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-family:'arial,sans-serif'; color:#000000;&quot; style=&quot; margin-top:12px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Help diagnose and report problems&lt;/li&gt;
&lt;li style=&quot; font-family:'arial,sans-serif'; color:#000000;&quot; style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Suggest fixes by sending in patches&lt;/li&gt;
&lt;li style=&quot; font-family:'arial,sans-serif'; color:#000000;&quot; style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Improve the code&lt;/li&gt;
&lt;li style=&quot; font-family:'arial,sans-serif'; color:#000000;&quot; style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Implement new features&lt;/li&gt;
&lt;li style=&quot; font-family:'arial,sans-serif'; color:#000000;&quot; style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Publish an article on your blog (send the link) or elsewhere (e.g., community sites) to tell the world about Raptor&lt;/li&gt;
&lt;li style=&quot; font-family:'arial,sans-serif'; color:#000000;&quot; style=&quot; margin-top:0px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Help with missing documentation.&lt;/li&gt;
&lt;li style=&quot; font-family:'arial,sans-serif'; color:#000000;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;etc...&lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; color:#000000; background-color:#ffffff;&quot;&gt;However, if you haven't got enough time but you still want to contribute, you can support Raptor development by making a donation :&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;a href=&quot;http://code.google.com/p/raptor/wiki/Donate&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#0057ae;&quot;&gt;Go to Raptor Donation Page&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <attribute name="title">
       <string comment="tab donations">Donations</string>
      </attribute>
     </widget>
     <widget class="QTextBrowser" name="tbLinks">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <attribute name="title">
       <string comment="tab links">Links</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>321</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="button OK">OK</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>UIAbout</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
